theme/brb/views/lib/extended_footer.hbs

```diff
@@ -4,10 +4,10 @@
 			<div class="col-md-4">
 				<h4>Kontakt</h4>
 				<ul>
-					<li><a href="mailto:brandenburg@schul-cloud.org" target="_blank" rel="noopener">Allgemein: brandenburg@schul-cloud.org</a></li>
-					<li><a href="mailto:inhalte@schul-cloud.org" target="_blank" rel="noopener">Inhalteanbieter: inhalte@schul-cloud.org</a></li>
-					<li><a href="mailto:datenschutz@schul-cloud.org" target="_blank" rel="noopener">Datenschutz: datenschutz@schul-cloud.org</a>
-					</li>
+					<li><a href="mailto:brandenburg@schul-cloud.org" target="_blank" rel="noopener">Allgemein: <span class="text-nowrap">brandenburg@schul-cloud.org</span></a></li>
+					<li><a href="mailto:feedback@schul-cloud.org" target="_blank" rel="noopener">Support-Anfragen: <span class="text-nowrap">feedback@schul-cloud.org</span></a></li>
+					<li><a href="mailto:inhalte@schul-cloud.org" target="_blank" rel="noopener">Inhalteanbieter: <span class="text-nowrap">inhalte@schul-cloud.org</span></a></li>
+					<li><a href="mailto:datenschutz@schul-cloud.org" target="_blank" rel="noopener">Datenschutz: <span class="text-nowrap">datenschutz@schul-cloud.org</span></a></li>
```

Comment:
can we move L8, L9, and L10 into a separate hbs and reuse it for open and extended?